### PR TITLE
Pioneer 17 precruise deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Pre-cruise deploy csv updates for Pioneer 17 (with dummy deploy date/time, lat/lon, depths)

Also - almost forgot...the MOPAK on CNSM-11 and CNSM-13 deployments was corrected to 11841 (discovered after recovery of CNSM-13 that we had the incorrect MOPAK SN).  Updated in associated configs in July so you can confirm via those.

### CODE REVIEW

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -475,3 +475,37 @@ AR52,,,,CP01CNSM-SBD12-04-PCO2AA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014
...
+AR61,,,,CP01CNSM-SBD11-01-MOPAK0000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-MOPAK0-11841,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD11-02-HYDGN0000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-HYDGN0-11407,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD11-06-METBKA000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-METLGR-00007,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-00-DCLENG000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGCON-BDCL12-00009,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-03-HYDGN0000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-HYDGN0-49860,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-04-PCO2AA000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-PCO2AA-33177,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-05-WAVSSA000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-WAVSSA-04571,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-06-METBKA000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-METLGR-00003,40.1,-70.7,,0,135,,,
+AR61,,,,CP01CNSM-SBD12-08-FDCHPA000,15,1,2021-10-26T00:00:00,,CGMCP-01CNSM-00015,,CGINS-FDCHPB-03904,40.1,-70.7,,-5,135,,,
```
Comment (asmith-whoi) [inline deployment/CP01CNSM_Deploy.csv:511]: OK

File: deployment/CP02PMCI_Deploy.csv [hunk 2]
```diff
@@ -134,3 +134,12 @@ AR52,,,,CP02PMCI-WFP01-02-DOFSTK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015
...
+AR61,,,,CP02PMCI-SBS01-00-STCENG000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,,CGCON-BSTC01-00040,40.2,-70.8,,0,127,,,
+AR61,,,,CP02PMCI-SBS01-01-MOPAK0000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,,CGINS-MOPAK0-58484,40.2,-70.8,,0,127,,,
+AR61,,,,CP02PMCI-WFP01-00-WFPENG000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGCON-CWFPCP-10302,40.2,-70.8,,104,127,,,
+AR61,,,,CP02PMCI-WFP01-01-VEL3DK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-VEL3DK-00037,40.2,-70.8,,104,127,,,
+AR61,,,,CP02PMCI-WFP01-02-DOFSTK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-DOFSTK-02724,40.2,-70.8,,104,127,,,
+AR61,,,,CP02PMCI-WFP01-03-CTDPFK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.2,-70.8,,104,127,,,
+AR61,,,,CP02PMCI-WFP01-04-FLORTK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.2,-70.8,,104,127,,,
+AR61,,,,CP02PMCI-WFP01-05-PARADK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.2,-70.8,,104,127,,,
```
Comment (asmith-whoi) [inline deployment/CP02PMCI_Deploy.csv:145]: OK

File: deployment/CP02PMCO_Deploy.csv [hunk 3]
```diff
@@ -134,3 +134,12 @@ AR52,,,,CP02PMCO-WFP01-02-DOFSTK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015
...
+AR61,,,,CP02PMCO-SBS01-00-STCENG000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,,CGCON-BSTC01-00010,40.1,-70.8,,0,148,,,
+AR61,,,,CP02PMCO-SBS01-01-MOPAK0000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,,CGINS-MOPAK0-11366,40.1,-70.8,,0,148,,,
+AR61,,,,CP02PMCO-WFP01-00-WFPENG000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.1,-70.8,,127,148,,,
+AR61,,,,CP02PMCO-WFP01-01-VEL3DK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.1,-70.8,,127,148,,,
+AR61,,,,CP02PMCO-WFP01-02-DOFSTK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.1,-70.8,,127,148,,,
+AR61,,,,CP02PMCO-WFP01-03-CTDPFK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-CTDPFK-00128,40.1,-70.8,,127,148,,,
+AR61,,,,CP02PMCO-WFP01-04-FLORTK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-FLORTK-01101,40.1,-70.8,,127,148,,,
+AR61,,,,CP02PMCO-WFP01-05-PARADK000,16,1,2021-10-26T00:00:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-PARADK-20466,40.1,-70.8,,127,148,,,
```
Comment (asmith-whoi) [inline deployment/CP02PMCO_Deploy.csv:145]: OK

File: deployment/CP02PMUO_Deploy.csv [hunk 4]
```diff
@@ -152,3 +152,12 @@ AR52,,,,CP02PMUO-WFP01-02-DOFSTK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017
...
+AR61,,,,CP02PMUO-SBS01-00-STCENG000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,,CGCON-BSTC01-00005,39.9,-70.7,,0,453,,,
+AR61,,,,CP02PMUO-SBS01-01-MOPAK0000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,,CGINS-MOPAK0-11358,39.9,-70.7,,0,453,,,
+AR61,,,,CP02PMUO-WFP01-00-WFPENG000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.9,-70.7,,429,453,,,
+AR61,,,,CP02PMUO-WFP01-01-VEL3DK000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.9,-70.7,,429,453,,,
+AR61,,,,CP02PMUO-WFP01-02-DOFSTK000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.9,-70.7,,429,453,,,
+AR61,,,,CP02PMUO-WFP01-03-CTDPFK000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.9,-70.7,,429,453,,,
+AR61,,,,CP02PMUO-WFP01-04-FLORTK000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.9,-70.7,,429,453,,,
+AR61,,,,CP02PMUO-WFP01-05-PARADK000,18,1,2021-10-26T00:00:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39.9,-70.7,,429,453,,,
```
Comment (asmith-whoi) [inline deployment/CP02PMUO_Deploy.csv:163]: OK

File: deployment/CP03ISPM_Deploy.csv [hunk 5]
```diff
@@ -35,3 +35,12 @@ AR48,,AR52,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31
...
+AR61,,,,CP03ISPM-SBS01-00-STCENG000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,,CGCON-BSTC01-00018,40.3,-70.8,,0,92,,,
+AR61,,,,CP03ISPM-SBS01-01-MOPAK0000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,,CGINS-MOPAK0-58481,40.3,-70.8,,0,92,,,
+AR61,,,,CP03ISPM-WFP01-00-WFPENG000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.3,-70.8,,68,92,,,
+AR61,,,,CP03ISPM-WFP01-01-VEL3DK000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.3,-70.8,,68,92,,,
+AR61,,,,CP03ISPM-WFP01-02-DOFSTK000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.3,-70.8,,68,92,,,
+AR61,,,,CP03ISPM-WFP01-03-CTDPFK000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.3,-70.8,,68,92,,,
+AR61,,,,CP03ISPM-WFP01-04-FLORTK000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.3,-70.8,,68,92,,,
+AR61,,,,CP03ISPM-WFP01-05-PARADK000,5,1,2021-10-26T00:00:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40.3,-70.8,,68,92,,,
```
Comment (asmith-whoi) [inline deployment/CP03ISPM_Deploy.csv:46]: OK

File: deployment/CP03ISSM_Deploy.csv [hunk 6]
```diff
@@ -403,3 +403,34 @@ AR52,,,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013
...
+AR61,,,,CP03ISSM-RID27-04-DOSTAD000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-DOSTAD-00410,40.3,-70.8,,7,93,,,
+AR61,,,,CP03ISSM-SBC11-00-CPMENG000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGCON-BCPM01-50004,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-00-DCLENG000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGCON-BDCL11-50004,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-01-MOPAK0000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-MOPAK0-11839,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-02-HYDGN0000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-HYDGN0-41692,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-06-METBKA000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-METLGR-00033,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-00-DCLENG000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGCON-BDCL12-50004,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-03-HYDGN0000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-HYDGN0-49868,40.3,-70.8,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-04-PCO2AA000,14,1,2021-10-26T00:00:00,,CGMCP-03ISSM-00014,,CGINS-PCO2AA-33182,40.3,-70.8,,0,93,,,
```
Comment (asmith-whoi) [inline deployment/CP03ISSM_Deploy.csv:436]: OK

File: deployment/CP04OSPM_Deploy.csv [hunk 7]
```diff
@@ -118,3 +118,12 @@ AR52,,,,CP04OSPM-WFP01-02-DOFSTK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014
...
+AR61,,,,CP04OSPM-SBS01-00-STCENG000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,,CGCON-BSTC01-00004,39.9,-70.8,,0,438,,,
+AR61,,,,CP04OSPM-SBS11-02-MOPAK0000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,,CGINS-MOPAK0-11357,39.9,-70.8,,0,438,,,
+AR61,,,,CP04OSPM-WFP01-00-WFPENG000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.9,-70.8,,417,438,,,
+AR61,,,,CP04OSPM-WFP01-01-VEL3DK000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.9,-70.8,,417,438,,,
+AR61,,,,CP04OSPM-WFP01-02-DOFSTK000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.9,-70.8,,417,438,,,
+AR61,,,,CP04OSPM-WFP01-03-CTDPFK000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.9,-70.8,,417,438,,,
+AR61,,,,CP04OSPM-WFP01-04-FLORTK000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.9,-70.8,,417,438,,,
+AR61,,,,CP04OSPM-WFP01-05-PARADK000,15,1,2021-10-26T00:00:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.9,-70.8,,417,438,,,
```
Comment (asmith-whoi) [inline deployment/CP04OSPM_Deploy.csv:129]: OK

File: deployment/CP04OSSM_Deploy.csv [hunk 8]
```diff
@@ -404,3 +404,34 @@ AR52,,,,CP04OSSM-SBD12-00-DCLENG000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013
...
+AR61,,,,CP04OSSM-RID27-04-DOSTAD000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-DOSTAD-00126,39.9,-70.8,,7,448,,,
+AR61,,,,CP04OSSM-SBC11-00-CPMENG000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGCON-BCPM01-50002,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD11-00-DCLENG000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGCON-BDCL11-50002,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD11-01-MOPAK0000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-MOPAK0-44005,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD11-02-HYDGN0000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-HYDGN0-Z5974,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD11-06-METBKA000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-METLGR-00038,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD12-00-DCLENG000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGCON-BDCL12-50002,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD12-03-HYDGN0000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-HYDGN0-Z5975,39.9,-70.8,,0,448,,,
+AR61,,,,CP04OSSM-SBD12-04-PCO2AA000,14,1,2021-10-26T00:00:00,,CGMCP-04OSSM-00014,,CGINS-PCO2AA-33183,39.9,-70.8,,0,448,,,
```
Comment (asmith-whoi) [inline deployment/CP04OSSM_Deploy.csv:437]: OK

**Review by asmith-whoi — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 17 AM Updates [DT-151]](https://trello.com/c/LYsz2Yeu/151-pioneer-17-am-updates-dt-151)

**Pull request merged**